packages/text-annotator-react/src/TextAnnotatorPopup.tsx

```diff
@@ -56,17 +55,17 @@ export const TextAnnotatorPopup = (props: TextAnnotationPopupProps) => {
   const dismiss = useDismiss(context);
 
   const role = useRole(context, { role: 'tooltip' });
-  
+
   const { getFloatingProps } = useInteractions([dismiss, role]);
 
   const selectedKey = selected.map(a => a.annotation.id).join('-');
 
   useEffect(() => {
-    // Ignore all selection changes except those accompanied by a pointer event.
+    // Ignore all selection changes except those accompanied by a user event.
     if (event) {
-      setOpen(selected.length > 0 && event.type === 'pointerup');
+      setOpen(selected.length > 0 && (event.type === 'pointerup' || event.type === 'keydown'));
```

Comment:
We should show the popup immediately once the user starts selection. That's because:
1. Prevents having inconsistent workarounds to handle the `up` event for the system shortcuts like `Ctrl + A`, as it can be captured only on `keydown`.
2. Behavior is consistent with the annotating tool of the popular Slab document-creation site